docs/website/packages/codegen/copy.md

```diff
@@ -0,0 +1,46 @@
+---
+sidebar_position: 2
+title: Copying
+description: codegen's copy function
+---
+
+The `copy` function is designed to facilitate the duplication of source code from one FHIR version to another.
+
+## Basic Example
+
+### Set Up Your Source and Target Directories
+
+Ensure that you have two directories: one for the source FHIR version and another for the target FHIR version. For example, you might have a directory structure like this:
+
+- `src/r4b` (source)
+- `src/r5` (target)
+
+### Prepare Conditional Preprocessing
+
+If you need conditional preprocessing (such as including code only for specific FHIR versions), add preprocessing directives to your source files. For example:
+
+```ts
+// #if fhir >= r5
+// Specific code for FHIR version r5 or higher
+```
+
+### Create a Copy Script
+
+Next, add a script to your `package.json` to use the `copy` function. Here's an example:
+
+```json
+...
+  "scripts": {
+    "copyfiles": "pnpm --dir=\"${PWD}/../codegen\" dev copy --source-fhir r4b --target-fhir r5 --source \"${PWD}/src/r4b\" --target \"${PWD}/src/r5\"",
+    ...
+  }
+...
+```
+
+- Adjust the `--source-fhir` and `--target-fhir` flags to match your source and target FHIR versions.
+- Specify the correct paths for your source and target directories in the `--source` and `--target` commands.
+
+### Run your Script
+
+Execute the script within the package where you created it using `pnpm copyfiles`.
+Check your target directory (e.g., src/r5) to ensure that the files have been copied and preprocessed as expected. Imported FHIR version paths in these files should be updated to the target version (ex. e.g. `import "@bonfhir/core/r5"` -> `"@bonfhir/core/r4b"`), and conditional preprocessing directives should have been correctly applied.
```

Comment:
I think this should be more proeminent as this is the main feature of the tool and the reason why we don't use a traditional copy tool.